app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -335,6 +340,25 @@ class BrowserFragment : Fragment(), BackHandler {
         }
     }
 
+    private fun assignSitePermissionsRules() {
```

Comment:
To align with the setting we could also rename this to `assignRecommendedSettings`

Reply:
👍

Reply:
Sorry, I have to keep the name `assignSitePermissionsRules`, because inside I will make the decision of which configuration will be assigned the **recommended** or **custom** settings.